<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Research scientist Inria Bordeaux, France
">
    <meta name="author" content="Clémence Frioux">
    <title>misc</title>
    <link rel="icon" type="image/png" href="/images/favicon.png" />
    <!-- Bootstrap -->
    <link rel="stylesheet" href="/css/main.css">
    <link rel="canonical" href="https://cfrioux.github.io/misc/">
    <!-- Font Awesome -->
    <link rel="stylesheet" href="/fontawesome-free-6.4.0-web/css/all.css">
    <meta name="google-site-verification" content="MJbvweFXM5RbkeipDYz3IhNhyEy4TinndcIhsewN4eI" />
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
        integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin="" />

    <!-- management of blocked Javascript -->
    <noscript>
        <style> .ifjavascript {
            display: none
        } </style>
    </noscript>
</head>

  <body>
    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
                    aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/index.html">Clémence Frioux</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
                <li >
                    <a class="navbar-link" href="/index.html">home</a>
                </li>
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                <li >
                    <a class="navbar-link" href="/research/">research</a>
                </li>
                
                
                
                <li class="active">
                    <a class="navbar-link" href="/misc/">misc</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/bio/">bio</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/news/news/">news</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/teaching/teaching/">teaching</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/outreach/outreach/">outreach</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/dni/">equality</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/contact/">contact</a>
                </li>
                
                
                <!--
                <li>
                    <a class="navbar-link" href="/index.html#contact">contact</a>
                </li>
                -->
            </ul>
        </div><!--/.nav-collapse -->
    </div>
</nav>

    <div class="container">
      <div class="starter-template">
    <h1>Miscellaneous</h1>
</div>

<h2><span class="text-primary">Supervising</span></h2>

<h3><span class="text-primary">Current supervising</span></h3>

<h4>Postdoctoral researchers</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2025 - 2027</dt>
    <dd>
        <span class="text-primary"><strong>Olivia Bulka</strong></span><br/>
        Metabolic modelling for non-culturable bacteria (INRAE EXPLOR'AE TARGET).<br/>
        <em>Co-supervision with <a href="https://bfp.bordeaux-aquitaine.hub.inrae.fr/personnel/pages-web-personnelles/prigent-sylvain" target="_blank">Sylvain Prigent</a> (INRAE, BFP)</em>
    </dd>
    
</dl>

<h4>PhD students</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2022 - 2025</dt>
    <dd>
        <span class="text-primary"><strong>Chabname Ghassemi Nedjad</strong></span><br/>
        Combinatorial problem solving for the detection of seeds in metabolic networks (Univ. Bordeaux).<br/>
        <em>co-director (50%) with <a href="https://loicpauleve.name" target="_blank">Loïc Paulevé</a> (CNRS, LaBRI)</em>
    </dd>
    
    <dt>2023 - 2026</dt>
    <dd>
        <span class="text-primary"><strong>Nicolas Maurice</strong></span><br/>
        Metagenomic assembly in complex microbial communities (PEPR Mistic, Inria, INRAE).<br/>
        <em>Co-supervisor (33%) with <a href="https://rvicedomini.github.io" target="_blank">Riccardo Vicedomini</a> (CNRS, IRISA-Inria Rennes) and <a href="https://people.rennes.inria.fr/Claire.Lemaitre/" target="_blank">Claire Lemaitre</a> (Inria Rennes)</em>
    </dd>
    
    <dt>2024 - 2027</dt>
    <dd>
        <span class="text-primary"><strong>Coralie Muller</strong></span><br/>
        Generation of metabolomic informed models of metabolisms in microbial communities (PEPR Mistic, Inria, INRAE).<br/>
        <em>co-director (50%) with <a href="https://bfp.bordeaux-aquitaine.hub.inrae.fr/personnel/pages-web-personnelles/prigent-sylvain" target="_blank">Sylvain Prigent</a> (INRAE BFP)</em>
    </dd>
    
    <dt>2024 - 2027</dt>
    <dd>
        <span class="text-primary"><strong>Mathilde Sola</strong></span><br/>
        Characterization of "gut microbiome-diet-health" relationships in humans through combined approaches of computational microbial ecology, metabolic modeling, and AI (INRAE-Inria).<br/>
        <em>Co-supervisor (35%) with <a href="https://scholar.google.com/citations?user=0UwkFsIAAAAJ&hl" target="_blank">Magali Berland</a> (INRAE MGPS) and <a href="https://scholar.google.com/citations?user=BIgHzvAAAAAJ&hl" target="_blank">Patrick Veiga</a> (INRAE MICALIS)</em>
    </dd>
    
</dl>



<h3><span class="text-primary">Past supervising</span></h3>

<h4>PhD students</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2020 - 2024</dt>
    <dd>
        <span class="text-primary"><strong>Maxime Lecomte</strong></span><br/>
        Hybrid approach for explainable metabolic modelling of microbial ecosystems (<a href="https://theses.hal.science/tel-04583312">HAL</a>) (Inria-INRAE)<br/>
        <em>co-supervisor (50%), co-supervised by Hélène Falentin (INRAE STLO) and David Sherman (Inria)</em>
    </dd>
    
</dl>

<h4>Engineers</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2023 - 2025</dt>
    <dd>
        <span class="text-primary"><strong>Léonard Brindel</strong></span><br/>
        Visualisation of metabolic potential screening in microbial communities<br/>
        <em>Inria ADT engineer</em>
    </dd>
    
    <dt>2022 - 2024</dt>
    <dd>
        <span class="text-primary"><strong>Coralie Muller</strong></span><br/>
        Inria exploratory action <a href="https://www.inria.fr/en/slimmest" target="_blank">Slimmest</a><br/>
        <em>Spatio-temporal modelling of microbial communities</em>
    </dd>
    
    <dt>2020 - 2022</dt>
    <dd>
        <span class="text-primary"><strong>Ariane Badoual</strong></span><br/>
        Inria ADT engineer<br/>
        <em>Metagenomic data treatment</em>
    </dd>
    
</dl>

<h4>Interns</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2024 - 2025</dt>
    <dd>
        <span class="text-primary"><strong>Juliette Audemard</strong></span><br/>
        Master 2 (apprenticeship) <a href="https://odf.u-paris.fr/fr/offre-de-formation/master-XB/sciences-technologies-sante-STS/bio-informatique-K2VO60MU/master-bioinformatique-parcours-biologie-informatique-IJTUMHHO.html" target="_blank">Bio-informatique</a>, Univ. Paris Cité<br/>
        <em>Metabolic modelling of cyanobacteria and their heterotrophic bacterial communities in the context of freshwater blooms (INRAE HOLOFLUX project led by Binta Diémé & Nicolas Creusot).</em>
    </dd>
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Killian Duguépéroux</strong></span><br/>
        Bachelor (7 weeks)  <a href="https://informatique.ens-lyon.fr/en" target="_blank">Computer science, ENS LYon</a><br/>
        <em>Dimensionality reduction of microbiome data. Co-supervised with <a href="https://scholar.google.com/citations?user=F1_9N4EAAAAJ&hl" target="_blank">Guilhem Sommeria-Klein</a>.</em>
    </dd>
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Juliette Audemard</strong></span><br/>
        Master 1 (4 months) <a href="https://odf.u-paris.fr/fr/offre-de-formation/master-XB/sciences-technologies-sante-STS/bio-informatique-K2VO60MU/master-bioinformatique-parcours-biologie-informatique-IJTUMHHO.html" target="_blank">Bio-informatique</a>, Univ. Paris Cité<br/>
        <em>Metabolic modelling of the phycosphere (INRAE HOLOFLUX project led by Binta Diémé & Nicolas Creusot).</em>
    </dd>
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Juliette Gronier</strong></span><br/>
        Bachelor (6 weeks)  <a href="https://informatique.ens-lyon.fr/en" target="_blank">Computer science, ENS Lyon</a><br/>
        <em>Numerical metabolic modelling of microbial communities</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Nicolas Maurice</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with Claire Lemaitre (Inria)<br/>
        <em>Metagenomic assembly in complex ecosystems</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Léonard Brindel</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with Franck Salin (INRAE)<br/>
        <em>Metagenomic data analysis</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Chabname Ghassemi Nedjad</strong></span><br/>
        Master 2 (6 months)  <a href="https://odf.u-paris.fr/fr/offre-de-formation/master-XB/sciences-technologies-sante-STS/bio-informatique-K2VO60MU/master-bioinformatique-parcours-biologie-informatique-IJTUMHHO.html" target="_blank">Bio-informatique</a>, Univ. Paris Cité<br/>
        <em>Metabolic modelling for large metagenomic studies</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Mathieu Bolteau</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux<br/>
        <em>Reverse ecology of microbial communities</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Maxime Lecomte</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with  <a href="https://biogeco.hub.inrae.fr/umr-biogeco/personnel/labarthe-simon" target="_blank">Simon Labarthe</a> (INRAE)<br/>
        <em>Metabolic modelling of a cheese bacterial community</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Ariane Badoual</strong></span><br/>
        Master 2 (6 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1<br/>
        <em>Modelling competition in microbial communities</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Margot Wagner</strong></span><br/>
        Master 2 (6 months)  <a href="https://osur.univ-rennes1.fr/MODE/" target="_blank">Modélisation en écologie</a>, Univ. de Rennes 1<br/>
        <em>Analysis of brown algae and microbiota's physiological response to varying growth conditions.</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Enora Fremy</strong></span><br/>
        Master 2 (6 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1<br/>
        <em>In silico prediction of brown-algal bacterial interactions</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Claire Lippold</strong></span><br/>
        Master 1 (3 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1. Co-supervised with Camille Trottier (Univ. Nantes)<br/>
        <em>Exploration and characterization of the microbiome associated to <em>Ectocarpus subulatus</em> str. BFT</em>
    </dd>
    
</dl>



<div class="row">
    <div class="col-md-6">
        <h2><span class="text-primary">Collective responsibilities</span></h2>
        
<h4>Scientific</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Co-head of the organisation committee of the French Conference of Bioinformatics  <a href="https://jobim2025.labri.fr">JOBIM 2025</a>  (> 500 attendees)</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Proceedings Program Committee of International Conference on Intelligent Systems for Molecular Biology</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Co-organisation of a <a href="https://jobim2022.sciencesconf.org/resource/page/id/7">mini-symposium</a> during the annual French bioinformatics conference (2022)</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2021-2022</dt>
    <dd>
        <span class="text-primary"><strong>International Conference on Computational Methods in Systems Biology (CMSB) - steering committee (2021), programme committee (2021, 2022)</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
</dl>

<h4>Institutional</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2023-..</dt>
    <dd>
        <span class="text-primary"><strong>Member of the Commission des Emplois de Recherche of Inria Bordeaux, a committee that assesses the scientific quality of researchers’ applications prior their recruitment</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2023-..</dt>
    <dd>
        <span class="text-primary"><strong>Member of the national committee for gender equality and inclusion at Inria</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2022-..</dt>
    <dd>
        <span class="text-primary"><strong>Member and co-founder of a local committee for gender equality, diversity and in- clusion at Inria Bordeaux</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
    <dt>2020-2022</dt>
    <dd>
        <span class="text-primary"><strong>Member of the Inria Bordeaux local committee for sustainable development</strong></span><br/>
        <br/>
        <em></em>
    </dd>
    
</dl>


    </div>
    <div class="col-md-6">
        <h2><span class="text-primary">PhD jury</span></h2>
        
<h4>Jury member</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Mathieu Bolteau</strong></span><br/>
        Nantes Univ.<br/>
        <em></em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Clémence Jospeh</strong></span><br/>
        KU Leuven (Belgium)<br/>
        <em></em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Marie Burel</strong></span><br/>
        Univ. Paris-Saclay<br/>
        <em></em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Déborah Boyenval</strong></span><br/>
        Univ. Côte d'Azur<br/>
        <em></em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Antoine Régimbeau</strong></span><br/>
        Nantes Univ.<br/>
        <em></em>
    </dd>
    
</dl>


    </div>
</div>

<h2><span class="text-primary">Scientific dissemination</span></h2>

<h4>Selection of talks in (relatively) recent selective conferences</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<p class="text-muted">
    This list also contains talks presented by students, in which case their names are indicated.
</p>

<dl class="dl-horizontal">
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Seed inference in interacting microbial communities using combinatorial optimization</strong></span><br/>
        <a href="https://cmsb2025.sciencesconf.org" target="_blank">23rd International conference on Computational Methods in Systems Biology (CMSB 2025)</a> -- Lyon, France<br/>
        <em>Talk by Chabname Ghassemi Nedjad</em>
    </dd>
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Metagenome-scale metabolic modelling for the characterization of cross-feeding interactions in freshwater cyanobacteria-associated microbial communities</strong></span><br/>
        <a href="https://jobim2025.labri.fr" target="_blank">JOBIM 2025</a> -- Bordeaux, France -- Best talk of the conference award<br/>
        <em>Talk by Juliette Audemard</em>
    </dd>
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Assessing assembly quality in metagenomes of increasing complexity sequenced with HiFi long reads</strong></span><br/>
        <a href="https://coursesandconferences.wellcomeconnectingscience.org/event/genome-informatics-20241113/" target="_blank">Genome Informatics 2024</a> -- Hinxton, United Kingdom<br/>
        <em>Talk by Nicolas Maurice (<a href="https://hal.science/hal-04822870v1" target="_blank">HAL</a>)</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Modelling the dynamics of Salmonella infection in the gut at the bacterial and host levels</strong></span><br/>
        <a href="https://asb-conference.hki-jena.de/sbmi-2023" target="_blank">7th International Symposium on Systems Biology of Microbial Infection</a> -- Jena, Germany<br/>
        <em>Talk by Coralie Muller and Pablo Ugalde Salas (<a href="https://inria.hal.science/hal-04229116" target="_blank">HAL</a>)</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        Computational Methods in Systems Biology 2021 -- Bordeaux, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        Metabolic Pathway Analysis 2021 (virtual)<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        JOBIM Conference (virtual), France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="http://eccb18.org/" target="_blank">European Conference on Computational Biology, ECCB 2018</a> -- September 8th-12th 2018, Athens, Greece<br/>
        <em>Talk - <a href="https://data-access.cesgo.org/index.php/s/uNR72XZ1nVG0kf8" target="_blank">Slides</a></em>
    </dd>
    
</dl>

<h4>Selection of (relatively) recent invited seminars and conferences</h4>
<!-- if category.name is poster -->


<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Reasoning, statistical learning and numerical modelling for the characterization of microbial communities</strong></span><br/>
        <a href="https://www.irisa.fr" target="_blank">IRISA DKM department</a> -- Rennes, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2025</dt>
    <dd>
        <span class="text-primary"><strong>Exploration of microbial communities, from compositional patterns to metabolic models</strong></span><br/>
        <a href="https://www.inria.cl/en" target="_blank">Inria Chile</a> -- Santiago, Chile<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2024</dt>
    <dd>
        <span class="text-primary"><strong>Exploration of microbial ecosystems - from compositional patterns to metabolic models</strong></span><br/>
        <a href="https://jobim2024.sciencesconf.org/resource/page/id/6" target="_blank"> JOBIM 20024 </a> -- Toulouse, France<br/>
        <em>Keynote</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Exploration of microbial ecosystems, from compositional patterns to metabolic models</strong></span><br/>
        Plant Health Institute, Montpellier, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Exploration of microbial ecosystems, from compositional patterns to metabolic models</strong></span><br/>
        KU Leuven, Belgium<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Logic modelling of metabolism - from individual networks to communities</strong></span><br/>
        Bordeaux Bioinformatics Center seminar, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Logic modelling of metabolism - from individual networks to communities</strong></span><br/>
        French workshop on Metabolic Modelling, Bordeaux, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Mathemetical and logic modelling in systems biology, applications to the metabolism of ecosystems</strong></span><br/>
        CEMRACS 2021, Marseille, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Gap-filling metabolism in systems biology and microbial systems ecology</strong></span><br/>
        BioNum Seminar — LaBRI, February 2020, Bordeaux, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        <a href="https://quadram.ac.uk/" target="_blank"> Quadram Institute of Biosciences </a>, May 2019, Norwich, UK<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        <a href="https://www.qtb.hhu.de/" target="_blank"> Institute of Quantitative and Theoretical Biology, HHU </a>, January 2019, Düsseldorf, Germany<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        <a href="https://bioinfo.lri.fr" target="_blank"> Bioinformatics research group, LRI </a>, November 2018, Orsay, France<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions in microbiotas</strong></span><br/>
        <a href="http://biologia.uc.cl/es/cuerpo-academico/profesor/45" target="_blank">Departamento de Ecología, Facultad de Ciencias Biológicas, Pontificia Universidad Católica de Chile </a>, November 2018, Santiago, Chile<br/>
        <em>Seminar</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions in microbiotas</strong></span><br/>
        <a href="http://www.ican-institute.org/team/umr_s872team-7-nutriomics-nutrition-obesity-systemic-approaches/" target="_blank"> UMR Nutriomics, ICAN </a>, October 2018, Paris, France<br/>
        <em>Seminar</em>
    </dd>
    
</dl>

<h4>Posters</h4>
<!-- if category.name is poster -->


<!-- show a text that says the list is non-exhaustive -->
<p class="text-muted">
    This list is non-exhaustive and mostly refers to posters I presented myself. For a complete and recent list of posters that were presented by students or colleagues, please refer to <a href="https://inria.hal.science/search/index/?qa%5Bauth_t%5D%5B%5D=Cl%C3%A9mence+Frioux&rows=30&docType_s=POSTER">HAL</a>.
</p>

<!-- if category.name is "Selection of talks in (relatively) recent selective conferences" -->

<dl class="dl-horizontal">
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="https://alam.science/alam-2018/" target="_blank">XXIV Congreso Latinoamericano de Microbiologia, ALAM 2018</a>, November 13th-16th 2018, Santiago, Chile<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="http://eccb18.org/" target="_blank">European Conference on Computational Biology, ECCB 2018</a>, September 8th-12th 2018, Athens, Greece<br/>
        <em>Poster - <a href="https://data-access.cesgo.org/index.php/s/fAtbOuD1YGgnH34" target="_blank">PDF</a></em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Combinatorial approaches for interactions prediction and community selection in holobionts.</strong></span><br/>
        <a href="https://www.biogenouest.org/contenu/gen2bio/gen2bio-2018" target="_blank">Gen2Bio 2018</a>, March 2018, Rennes, France<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Combining size and exchange minimization enables scalable screening of microbial consortia</strong></span><br/>
        <a href="https://vibconferences.be/event/applied-bioinformatics-in-life-sciences-2nd-edition" target="_blank">Applied Bioinformatics in Life Sciences ABLS 2018</a>, March 8th-9th 2018, Leuven, Belgium<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Topology to elucidate the respective roles of species in a symbiotic community</strong></span><br/>
        <a href="https://symposium.inra.fr/holobiont-paris2017" target="_blank">International Conference on Holobionts</a>, April 19th-21st 2017, Paris<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2016</dt>
    <dd>
        <span class="text-primary"><strong>Combinatorial gap-filling for genome-scale metabolic networks using taxonomy and pathway information</strong></span><br/>
        École Jeunes Chercheurs et Jeunes Chercheuses CNRS – 3ème École thématique de modélisation formelle des réseaux de régulation biologique <a href="http://i3s.unice.fr/bioregul/" target="_blank">Bioregul</a>, June 6th-10th, Porquerolles<br/>
        <em>Poster</em>
    </dd>
    
</dl>



    </div>
    <div class="navbar navbar-default navbar-bottom">
    <div class="container">
        <h4>Clémence Frioux (she/her)</h4>
        <div class="row">
            <div class="col-md-4">
                <p>Research scientist Inria Bordeaux, France
</p>
            </div>
            <div class="col-md-3">
                <p>
                    
                    <a href="mailto:(firstname).(lastname)@inria.fr">
                        (firstname).(lastname)@inria.fr
                    </a><br/>
                    
                    
                </p>
            </div>
            <div class="col-md-3">
                <p>
                    
                    <i class="fa-brands fa-github-square"></i>
                    <a href="https://github.com/cfrioux">
                        cfrioux
                    </a><br/>
                    
                    
                    
                    <i class="fa-brands fa-linkedin"></i>
                    <a href="https://www.linkedin.com/in/clemencefrioux">
                        in/clemencefrioux
                    </a><br/>
                    
                    
                    <i class="fas fa-graduation-cap"></i>
                    <a href="https://scholar.google.com/citations?user=ILXDngoAAAAJ">
                        Clémence Frioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-mastodon"></i>
                    <a href="https://mamot.fr/@clemencefrioux">
                        clemencefrioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-orcid"></i>
                    <a href="https://orcid.org/0000-0003-2114-0697">
                        0000-0003-2114-0697
                    </a><br/>
                    
                    
                    
                    <i class="fa-solid fa-building-columns"></i>
                    <a href="https://cv.hal.science/clemence-frioux">
                        idHAL clemence-frioux
                    </a><br />
                    
                </p>
            </div>
        </div>
        <div class="footer-col-wrapper">
            <div class="footer-col-thanks">
                <p>
                    pages made with <a href="http://jekyllrb.com">Jekyll</a>,
                    <a class="button" href="https://github.com/cfrioux/cfrioux.github.io">
                        <button class="btn btn-xs btn-primary"><i
                                class="fa fa-download"></i> get source code
                        </button>
                    </a>
                </p>
            </div>
        </div>
    </div>
</div>

    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<!-- Include all compiled plugins (below), or include individual files as needed -->
<script src="/javascripts/bootstrap.min.js"></script>


<!-- Global Site Tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-115007039-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-115007039-1');
</script>


  </body>
</html>
