<p class="text-justify"></p>
I am a junior researcher (<i> "charg√©e de recherche" </i>) at the <a href="https://www.inria.fr/en/inria-centre-university-bordeaux" target="_blank">Inria centre at the University of Bordeaux</a>, France. My research is at the interface of computational biology, computer science and systems biology. My main objective is to develop methods and models for the analysis of microbial communities, and in particular to generate hypotheses on the functions they harbour and the interactions they establish. To that end, I am interested in <b>metagenomics</b>, <b>metabolic modelling</b> and <b>optimisation problem solving</b>. 
</p>

<p class="text-justify"></p>
Below I summarise some of my recent works. 
</p>

<p class="text-justify"></p>
Together with colleagues, notably the <a href="https://www.falk.science/" target="_blank">Hildebrand group</a> at the <a href="https://quadram.ac.uk/" target="_blank">Quadram Institute</a> and <a href="https://www.earlham.ac.uk/" target="_blank">Earlham Institute</a>, we proposed the concept of enterosignatures as a mathematical model describing the microbial guilds inhabiting the human gut microbiome. In our paper published in <i>Cell Host and Microbe</i> in 2023, we show that the gut microbiome variability can be accurately described by combinations of enterosignatures among five.  We demonstrate the relevance of the concept to compare microbiome compositions in individuals, study the changes over lifetime and after perturbations, and also detect atypical microbiome compositions. <a href="https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00217-2" target="_blank">[Read more]</a>.
</p>

<p class="text-justify"></p>
In a collaborative work around the PhD of Maxime Lecomte, we built metabolic models and analysed the dynamics of a three species bacterial community during cheese production. With this <i>digital twin</i> obtained using metabolic network refinement, dynamic flux balance analysis and omics data integration, we showed the contribution of the species in the production of aroma compounds. This works illustrates both the potential and the complexity of inferring the dynamics of metabolic interactions within microbial community at the metabolic level, even in small communities. <a href="https://www.biorxiv.org/content/10.1101/2023.05.05.539417v1" target="_blank">[Read more]</a>.  
</p>

<p class="text-justify"></p>
Numerical models of the metabolism are great, but they cannot always be applied. When focusing on screening the metabolic potential of large collections of microbial genomes, or large-size microbial communities, a simpler model can be useful to efficiently assess the putative roles of each species. To that end, a significant part of my research as been dedicated to use a discrete, Boolean-based, model of metabolic producibility as a proxy to understand the metabolism of species in their environment. I tackled this problem using combinatorial optimisation problem solving, for instance for <a href="https://github.com/cfrioux/miscoto" target="_blank"><i>minimal community selection</i></a> in a paper published in <i>Bioinformatics</i> in 2018. One of my recent projects published in <i>eLife</i> is the development of an all-in-one pipeline for the screening of large microbial communities using discrete metabolic modelling <a href="https://elifesciences.org/articles/61968" target="_blank">[Read more]</a>.
</p>

<!-- <p class="text-justify">
    I am interested in solving <b>combinatorial optimisation problems</b> using <a href="https://potassco.org/" target="_blank">Answer Set Programming</a> in a computational biology context. Such logic programming paradigm can apply to modelling the metabolism of organisms. Metabolic networks are obtained by inferring the presence of metabolic reactions based on the genomic content of the associated organism. Such networks are the basis of models that can be simulated using various mathematical frameworks among which flux balance analysis (FBA) or network expansion. The latter is notably perfectly suited to logic programming and is implemented in many software systems that I developped (see below). As an example, the <a href="https://github.com/cfrioux/menetools" target="_blank"><i>Menetools</i></a> package includes several tools enabling dedicated to exploring the producibility potential of metabolic networks.
</p>
<p class="text-justify">
    Optimisation problems are numerous in the field of metabolic modelling. A first one that I worked on is the refinement of metabolic network or <b>gap-filling</b>. It aims at selecting a limited number of metabolic reactions from databases in order to ensure a metabolic objective is met by the metabolic model. This is the purpose of the Python package <a href="https://github.com/bioasp/meneco" target="_blank"><i>Meneco</i></a> (<i>PLOS Computational Biology 2017</i>).
    In addition to Meneco, I created, in collaboration with scientists of Potsdam University, a <b>hybrid</b> method for metabolic gap-filling, enabling the reconciliation of flux balance analysis and network expansion modellings. This work is implemented in <a href="https://github.com/cfrioux/fluto" target="_blank"><i>Fluto</i></a> (<i>LPNMR Conference 2017, best Student Paper, extended version of the manuscript in Theory and Practice of Logic Programming 2019</i>). 
</p>

<p class="text-justify">
    I dedicated the past few years to applying optimisation problems and metabolic modelling to communities of organisms. <a href="https://github.com/cfrioux/miscoto" target="_blank"><i>MiSCoTo</i></a> is my first project on this topic. The underlying optimisation problem (<i>Bioinformatics 2018</i>) aims at <b>selecting minimal communities within a collection of metabolic networks</b> such that the selected communities meet a metabolic objective characterised by the producibility of metabolic compounds. MiSCoTo can in addition suggest metabolic exchanges that would be needed between the species of the selected communities.
</p>

<p class="text-justify">
    One of my latest projects if the development of <a href="https://github.com/AuReMe/metage2metabo" target="_blank"><i>Metage2Metabo</i></a> or M2M (<i>eLife 2020</i>), <b>an all-in-one workflow permitting the global analysis of a possibly large community of organisms in the context of microbiota</b>. M2M is a flexible pipeline with the following main steps: automation of metabolic network reconstruction with Pathway Tools, computation of individual and collective metabolic capabilities, estimation of the added-value of cooperation within the community, selection of minimal communities and <b>identification of key species</b> with respect to a metabolic objective. 
</p> -->
