<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Assistant Professor Université Paris Cité | Institut Jaques Monod Paris, France
">
    <meta name="author" content="Clémence Frioux">
    <title>misc</title>
    <link rel="icon" type="image/png" href="/images/favicon.png" />
    <!-- Bootstrap -->
    <link rel="stylesheet" href="/css/main.css">
    <link rel="canonical" href="http://localhost:4000/misc/">
    <!-- Font Awesome -->
    <link rel="stylesheet" href="/fontawesome-free-6.4.0-web/css/all.css">
    <meta name="google-site-verification" content="MJbvweFXM5RbkeipDYz3IhNhyEy4TinndcIhsewN4eI" />
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- management of blocked Javascript -->
    <noscript>
        <style> .ifjavascript {
            display: none
        } </style>
    </noscript>
</head>

  <body>
    <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
    <div class="container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar"
                    aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/index.html">Yves Clément</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
                <li >
                    <a class="navbar-link" href="/index.html">home</a>
                </li>
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                <li >
                    <a class="navbar-link" href="/research/">research</a>
                </li>
                
                
                
                <li class="active">
                    <a class="navbar-link" href="/misc/">misc</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/bio/">bio</a>
                </li>
                
                
                
                <li >
                    <a class="navbar-link" href="/contact/">contact</a>
                </li>
                
                
                <!--
                <li>
                    <a class="navbar-link" href="/index.html#contact">contact</a>
                </li>
                -->
            </ul>
        </div><!--/.nav-collapse -->
    </div>
</nav>

    <div class="container">
      <div class="starter-template">
    <h1>Miscellaneous</h1>
</div>

<div class="row">
    <div class="col-md-6">
        <h2><span class="text-primary">Scientific dissemination</span></h2>
        
<h3>Scientific communication</h3>
<dl class="dl-horizontal">
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Exploration of microbial ecosystems, from compositional patterns to metabolic models</strong></span><br/>
        Invited seminar, KU Leuven, Belgium<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Logic modelling of metabolism - from individual networks to communities</strong></span><br/>
        Bordeaux Bioinformatics Center seminar, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Logic modelling of metabolism - from individual networks to communities</strong></span><br/>
        French workshop on Metabolic Modelling, Bordeaux, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        Computational Methods in Systems Biology 2021, Bordeaux, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        Metabolic Pathway Analysis 2021 (virtual)<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Metabolic complementarity applied to the screening of microbiota and the identification of key species</strong></span><br/>
        JOBIM Conference (virtual), France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Gap-filling metabolism in systems biology and microbial systems ecology</strong></span><br/>
        BioNum Seminar — LaBRI, February 2020, Bordeaux, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        Invited seminar<a href="https://quadram.ac.uk/" target="_blank"> Quadram Institute of Biosciences </a>, May 2019, Norwich, UK<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        Invited research team seminar<a href="https://www.qtb.hhu.de/" target="_blank"> Institute of Quantitative and Theoretical Biology, HHU </a>, January 2019, Düsseldorf, Germany<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Investigating host-microbiota cooperation with gap-filling optimization problems</strong></span><br/>
        Invited research team seminar <a href="https://bioinfo.lri.fr" target="_blank"> Bioinformatics research group, LRI </a>, November 2018, Orsay, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions in microbiotas</strong></span><br/>
        Invited research team seminar <a href="http://biologia.uc.cl/es/cuerpo-academico/profesor/45" target="_blank">Departamento de Ecología, Facultad de Ciencias Biológicas, Pontificia Universidad Católica de Chile </a>, November 2018, Santiago, Chile<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions in microbiotas</strong></span><br/>
        <a href="http://eventos.cmm.uchile.cl/sysbio2018/" target="_blank">II Workshop on Systems Biology Chile-France</a>, November 7th-9th 2018, Valparaiso, Chile<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="https://alam.science/alam-2018/" target="_blank">XXIV Congreso Latinoamericano de Microbiologia, ALAM 2018</a>, November 13th-16th 2018, Santiago, Chile<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="http://eccb18.org/" target="_blank">European Conference on Computational Biology, ECCB 2018</a>, September 8th-12th 2018, Athens, Greece<br/>
        <em>Talk - <a href="https://data-access.cesgo.org/index.php/s/uNR72XZ1nVG0kf8" target="_blank">Slides</a></em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="http://eccb18.org/" target="_blank">European Conference on Computational Biology, ECCB 2018</a>, September 8th-12th 2018, Athens, Greece<br/>
        <em>Poster - <a href="https://data-access.cesgo.org/index.php/s/fAtbOuD1YGgnH34" target="_blank">PDF</a></em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions in microbiotas</strong></span><br/>
        Invited research team seminar <a href="http://www.ican-institute.org/team/umr_s872team-7-nutriomics-nutrition-obesity-systemic-approaches/" target="_blank"> UMR Nutriomics, ICAN </a>, October 2018, Paris, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Scalable and exhaustive screening of metabolic functions carried out by microbial consortia</strong></span><br/>
        <a href="https://js.univ-nantes.fr/navigation/cl10-systems-ecology-which-advances-towards-ecosystems-understanding--2184302.kjsp?RH=1486401620746" target="_blank">Journées scientifiques de Nantes 2018</a>, June 2018, Nantes, France<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Combinatorial approaches for interactions prediction and community selection in holobionts.</strong></span><br/>
        <a href="https://www.biogenouest.org/contenu/gen2bio/gen2bio-2018" target="_blank">Gen2Bio 2018</a>, March 2018, Rennes, France<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Combining size and exchange minimization enables scalable screening of microbial consortia</strong></span><br/>
        <a href="https://vibconferences.be/event/applied-bioinformatics-in-life-sciences-2nd-edition" target="_blank">Applied Bioinformatics in Life Sciences ABLS 2018</a>, March 8th-9th 2018, Leuven, Belgium<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Toward the study of metabolic functions in algal-bacterial holobionts</strong></span><br/>
        <a href="https://ge2017.sciencesconf.org/" target="_blank">4ème Colloque du GDR de Génomique Environnementale</a>, September 13-15th 2017, Marseille<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Topology to elucidate the respective roles of species in a symbiotic community</strong></span><br/>
        <a href="https://symposium.inra.fr/holobiont-paris2017" target="_blank">International Conference on Holobionts</a>, April 19th-21st 2017, Paris<br/>
        <em>Poster</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Hybrid gap-filling to reconcile qualitative and quantitative abstractions of metabolism</strong></span><br/>
        <a href="http://bioss-cnrs.fr/manif/jnbioss_201703/jnbioss201703.html" target="_blank">3ème édition des Journées annuelles du GDR BIOSS </a>, March 13-14th 2017, Montpellier<br/>
        <em>Talk</em>
    </dd>
    
    <dt>2016</dt>
    <dd>
        <span class="text-primary"><strong>Combinatorial gap-filling for genome-scale metabolic networks using taxonomy and pathway information</strong></span><br/>
        École Jeunes Chercheurs CNRS – 3ème École thématique de modélisation formelle des réseaux de régulation biologique <a href="http://i3s.unice.fr/bioregul/" target="_blank">Bioregul</a>, June 6th-10th, Porquerolles<br/>
        <em>Poster</em>
    </dd>
    
</dl>

<h3>Scientific popularizarion</h3>
<dl class="dl-horizontal">
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong><a href="https://interstices.info/modeliser-les-communautes-bacteriennes-pour-mieux-comprendre-leur-fonctionnement/" target="_blank"> Scientific popularisation article on the interest of modelling the metabolism of bacterial communities </a></strong></span><br/>
        Interstices<br/>
        <em>With Simon Labarthe</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong><a href="https://github.com/cfrioux/outreach" target="_blank"> Outreach activity on Metagenomics and Metabolic modelling </a></strong></span><br/>
        Open source activity.<br/>
        <em>Code and resources for an outreach activity.</em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong><a href="https://math-interactions.u-bordeaux.fr/Nos-projets-evenements/Moi-Informaticienne-Moi-Mathematicienne" target="_blank"> MIMM, moi informaticienne, moi mathématicienne </a></strong></span><br/>
        Bordeaux.<br/>
        <em>Free internship at the University of Bordeaux for young girls in 9th and 10th grade in order to encourage them to choose mathematics and computer science, allows them to discover training, research and jobs in these two disciplines.</em>
    </dd>
    
    <dt>2021 - ...</dt>
    <dd>
        <span class="text-primary"><strong><a href="https://chiche-snt.fr/" target="_blank"> Chiche ! Un ou une scientifique, une classe </a></strong></span><br/>
        Bordeaux and its surrounding regions.<br/>
        <em>Visit of a scientist in high schools in order to advertise academic careers and computer science research while deconstructing stereotypes.</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong><a href="https://norwichsciencefestival.co.uk/about/norwich-science-festival-2019/" target="_blank"> "Microbes of the gut, the good and the bad in us" </a></strong></span><br/>
        <a href="https://norwichsciencefestival.co.uk/" target="_blank">Norwich Science Festival</a>, October 2019 — Norwich, UK<br/>
        <em>Presentation of research on the gut microbiota by scientists of the <a href="https://quadram.ac.uk/" target="_blank">Quadram Institute</a> and the <a href="https://www.earlham.ac.uk/" target="_blank">Earlham Institute</a></em>
    </dd>
    
    <dt>2016</dt>
    <dd>
        <span class="text-primary"><strong><a href="http://dai.ly/x4a2r5b" target="_blank">Une petite histoire de symbiose(s)</a></strong></span><br/>
        Very Short Films Festival <a href="http://www.sciences-en-courts.fr/" target="_blank">Sciences en Court[s]</a><br/>
        <em>2nd jury prize</em>
    </dd>
    
    <dt>2016</dt>
    <dd>
        <span class="text-primary"><strong><a href="http://dai.ly/x4b068b" target="_blank">Une petite histoire de symbiose(s)</a></strong></span><br/>
        Short Film Festival <a href="https://jebif.fr/fr/2016/02/concours-video-jobim-2016/" target="_blank">Bioinfuse</a><br/>
        <em><a href="https://jobim2016.sciencesconf.org/" target="_blank">JOBIM 2016</a></em>
    </dd>
    
</dl>


    </div>
    <div class="col-md-6">
        <h2><span class="text-primary">Teaching</span></h2>
        
<h3></h3>
<dl class="dl-horizontal">
    
    <dt>2020-...</dt>
    <dd>
        <span class="text-primary"><strong>Master - Engineering school</strong></span><br/>
        <a href="https://enstbb.bordeaux-inp.fr/fr" target="_blank">ENSTBB (Bordeaux INP)</a><br/>
        <em>Bioinformatics</em>
    </dd>
    
    <dt>2022-...</dt>
    <dd>
        <span class="text-primary"><strong>Master - Engineering school</strong></span><br/>
        <a href="https://enseirb-matmeca.bordeaux-inp.fr/fr" target="_blank">ENSEIRB-MATMECA (Bordeaux INP)</a><br/>
        <em>Answer Set Programming</em>
    </dd>
    
    <dt>2020-2022</dt>
    <dd>
        <span class="text-primary"><strong>Graduate school</strong></span><br/>
        <a href="https://ed-mi.u-bordeaux.fr/" target="_blank">Doctoral school in computer science and mathematics</a>, Univ. Bordeaux<br/>
        <em>Answer Set Programming</em>
    </dd>
    
    <dt>2020-2021</dt>
    <dd>
        <span class="text-primary"><strong>Licence</strong></span><br/>
        Univ. Bordeaux<br/>
        <em>Python</em>
    </dd>
    
    <dt>2020-2021</dt>
    <dd>
        <span class="text-primary"><strong>Master</strong></span><br/>
        <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bioinformatics Master's degree</a>, Univ. de Bordeaux<br/>
        <em>Programming project supervision</em>
    </dd>
    
    <dt>2022-...</dt>
    <dd>
        <span class="text-primary"><strong>Licence</strong></span><br/>
        <a href="https://enseirb-matmeca.bordeaux-inp.fr/fr" target="_blank">ENSEIRB-MATMECA (Bordeaux INP)</a><br/>
        <em>Answer Set Programming</em>
    </dd>
    
    <dt>2017-2018</dt>
    <dd>
        <span class="text-primary"><strong>Licence</strong></span><br/>
        <a href="http://www.ensai.fr/" target="_blank">ENSAI</a><br/>
        <em>Algorithmique et programmation, LaTeX, suivi de projet Python, optimisation</em>
    </dd>
    
    <dt>2017-2018</dt>
    <dd>
        <span class="text-primary"><strong>Licence</strong></span><br/>
        <a href="https://www.univ-rennes1.fr/" target="_blank">Univ. de Rennes 1</a><br/>
        <em>Bureautique, programmation scientifique</em>
    </dd>
    
</dl>


    </div>
    <div class="col-md-6">
        <h2><span class="text-primary">Supervising</span></h2>
        
<h3>Engineer supervision</h3>
<dl class="dl-horizontal">
    
    <dt>2022 - 2024</dt>
    <dd>
        <span class="text-primary"><strong>Coralie Muller</strong></span><br/>
        Inria exploratory action <a href="https://www.inria.fr/en/slimmest" target="_blank">Slimmest</a><br/>
        <em>Spatio-temporal modelling of microbial communities</em>
    </dd>
    
    <dt>2020 - 2022</dt>
    <dd>
        <span class="text-primary"><strong>Ariane Badoual</strong></span><br/>
        Inria ADT engineer<br/>
        <em>Metagenomic data treatment</em>
    </dd>
    
</dl>

<h3>PhD supervision</h3>
<dl class="dl-horizontal">
    
    <dt>2020 - 2023</dt>
    <dd>
        <span class="text-primary"><strong>Maxime Lecomte</strong></span><br/>
        PhD student Inria-INRAE, co-supervised by H. Falentin (INRAE STLO) and D. Sherman (Inria).<br/>
        <em>Hybrid methods for numerical and discrete modelling of microbial communities</em>
    </dd>
    
    <dt>2022 - 2025</dt>
    <dd>
        <span class="text-primary"><strong>Chabname Ghassemi Nedjad</strong></span><br/>
        PhD student Univ. Bordeaux, co-supervised by L. Paulevé (CNRS, LaBRI).<br/>
        <em>Combinatorial problem solving for the detection of seeds in metabolic networks</em>
    </dd>
    
</dl>

<h3>Internship supervision</h3>
<dl class="dl-horizontal">
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Nicolas Maurice</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with C. Lemaitre (Inria)<br/>
        <em>Metagenomic assembly in complex ecosystems</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Léonard Brindel</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with F. Salin (INRAE)<br/>
        <em>Metagenomic data analysis</em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Chabname Ghassemi Nedjad</strong></span><br/>
        Master 2 (6 months)  <a href="https://odf.u-paris.fr/fr/offre-de-formation/master-XB/sciences-technologies-sante-STS/bio-informatique-K2VO60MU/master-bioinformatique-parcours-biologie-informatique-IJTUMHHO.html" target="_blank">Bio-informatique</a>, Univ. Paris Cité<br/>
        <em>Metabolic modelling for large metagenomic studies</em>
    </dd>
    
    <dt>2021</dt>
    <dd>
        <span class="text-primary"><strong>Mathieu Bolteau</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux<br/>
        <em>Reverse ecology of microbial communities</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Maxime Lecomte</strong></span><br/>
        Master 2 (6 months)  <a href="https://www.u-bordeaux.fr/formation/2018/PRMA_74/bio-informatique" target="_blank">Bio-informatique</a>, Univ. de Bordeaux. Co-supervised with S. Labarthe (INRAE)<br/>
        <em>Metabolic modelling of a cheese bacterial community</em>
    </dd>
    
    <dt>2020</dt>
    <dd>
        <span class="text-primary"><strong>Ariane Badoual</strong></span><br/>
        Master 2 (6 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1<br/>
        <em>Modelling competition in microbial communities</em>
    </dd>
    
    <dt>2019</dt>
    <dd>
        <span class="text-primary"><strong>Margot Wagner</strong></span><br/>
        Master 2 (6 months)  <a href="https://osur.univ-rennes1.fr/MODE/" target="_blank">Modélisation en écologie</a>, Univ. de Rennes 1<br/>
        <em>Analysis of brown algae and microbiota's physiological response to varying growth conditions.</em>
    </dd>
    
    <dt>2018</dt>
    <dd>
        <span class="text-primary"><strong>Enora Fremy</strong></span><br/>
        Master 2 (6 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1<br/>
        <em>In silico prediction of brown-algal bacterial interactions</em>
    </dd>
    
    <dt>2017</dt>
    <dd>
        <span class="text-primary"><strong>Claire Lippold</strong></span><br/>
        Master 1 (3 months) <a href="https://sve.univ-rennes1.fr/bio-informatique" target="_blank">Bio-Informatique et Génomique</a>, Univ. de Rennes 1. Co-supervised with C. Trottier (Univ. Nantes)<br/>
        <em>Exploration and characterization of the microbiome associated to <em>Ectocarpus subulatus</em> str. BFT</em>
    </dd>
    
</dl>


    </div>
    <div class="col-md-6">
        <h2><span class="text-primary">PhD jury</span></h2>
        
<h3>Jury member</h3>
<dl class="dl-horizontal">
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Clémence Jospeh</strong></span><br/>
        KU Leuven (Belgium)<br/>
        <em></em>
    </dd>
    
    <dt>2023</dt>
    <dd>
        <span class="text-primary"><strong>Marie Burel</strong></span><br/>
        Univ. Paris-Saclay<br/>
        <em></em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Déborah Boyenval</strong></span><br/>
        Univ. Côte d'Azur<br/>
        <em></em>
    </dd>
    
    <dt>2022</dt>
    <dd>
        <span class="text-primary"><strong>Antoine Régimbeau</strong></span><br/>
        Univ. Nantes<br/>
        <em></em>
    </dd>
    
</dl>


    </div>
</div>

    </div>
    <div class="navbar navbar-default navbar-bottom">
    <div class="container">
        <h4>Yves Clément (he/him)</h4>
        <div class="row">
            <div class="col-md-4">
                <p>Assistant Professor Université Paris Cité | Institut Jaques Monod Paris, France
</p>
            </div>
            <div class="col-md-3">
                <p>
                    
                    <a href="mailto:(firstname).(lastname)@inria.fr">
                        (firstname).(lastname)@inria.fr
                    </a><br/>
                    
                    
                </p>
            </div>
            <div class="col-md-3">
                <p>
                    
                    <i class="fa-brands fa-github-square"></i>
                    <a href="https://github.com/cfrioux">
                        cfrioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-twitter-square"></i>
                    <a href="https://twitter.com/c_frioux">
                        c_frioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-linkedin"></i>
                    <a href="https://www.linkedin.com/in/clemencefrioux">
                        in/clemencefrioux
                    </a><br/>
                    
                    
                    <i class="fas fa-graduation-cap"></i>
                    <a href="https://scholar.google.com/citations?user=ILXDngoAAAAJ">
                        Clémence Frioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-mastodon"></i>
                    <a href="https://mamot.fr/@clemencefrioux">
                        clemencefrioux
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-orcid"></i>
                    <a href="https://orcid.org/0000-0003-2114-0697">
                        0000-0003-2114-0697
                    </a><br/>
                    
                    
                    <i class="fa-brands fa-researchgate"></i>
                    <a href="https://www.researchgate.net/profile/Clemence-Frioux">
                        Clemence-Frioux
                    </a><br/>
                    
                </p>
            </div>
        </div>
        <div class="footer-col-wrapper">
            <div class="footer-col-thanks">
                <p>
                    pages made with <a href="http://jekyllrb.com">Jekyll</a>,
                    <a class="button" href="https://github.com/cfrioux/cfrioux.github.io">
                        <button class="btn btn-xs btn-primary"><i
                                class="fa fa-download"></i> get source code
                        </button>
                    </a>
                </p>
            </div>
        </div>
    </div>
</div>

    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<!-- Include all compiled plugins (below), or include individual files as needed -->
<script src="/javascripts/bootstrap.min.js"></script>


<!-- Global Site Tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-115007039-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-115007039-1');
</script>


  </body>
</html>
